src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportService;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportServiceImplTest {
+    private static ReportService reportService;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
```

Comment:
```suggestion
    public static void beforeClass() {
```
remove from everywhere